data-processing-lib/python/src/data_processing/test_support/launch/transform_test.py

```diff
@@ -21,7 +21,7 @@
 
 class AbstractTransformLauncherTest(AbstractTest):
     """
-    The launcher test class for all/most AbstractTransformLauncher implementations.
+    The Ray-based test class for all/most AbstractTransform implementations.
```

Comment:
You seem to be dropping things from dev.  This comment was more correct and the new ray-specific one is not.

Reply:
fixed